test/Assert.t.sol

```diff
@@ -25,4 +26,39 @@ contract TestAsserts is Test, HelperAssert {
         string memory reason = "Testing assertion";
         t(true, reason);
     }
-}
+
+    function test_HelperAssert_t_true() public {
+        t(true, "t does not revert for true"); 
+    }
+
+    function test_HelperAssert_t_false() public {
+        vm.expectRevert(PlatformTest.TestAssertFail.selector);
+        t(false, "t does not revert for true"); 
+    }
+
+    function test_HelperAssert_eq_x_x_concrete() public {
+        uint256 x = 1;
+        eq(x, x, "eq does not revert with equal values");
+    }
+
+    function testFuzz_HelperAssert_eq_x_x_fuzz(uint256 x) public {
+        eq(x, x, "eq does not revert with the fuzz values");
+    }
+
+    function test_HelperAssert_eq_x_y_concrete() public {
+        uint256 x = 2;
+        uint256 y = 4;
+        vm.expectEmit(true, true, false, false);
+        emit AssertEqFail("");
```

Comment:
shouldnt this fail? because the emitted event should have the string "eq reverts with different values"